docs/DeveloperGuide.md

```diff
@@ -105,6 +105,45 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case resumes at step 1.
 
+#### Use case: Remove a cheese
+
+**MSS**
+
+1. User enters the index of cheese to be removed.
+2. CHIM removes the cheese from the list of cheese.
+   
+   Use case ends.
+
+**Extensions**
+* 1a. The list is empty.
+    * 1a1.CHIM responds that there are no existing customers. 
```

Comment:
Would be nice to have spacing between CHIM and the full stop

Reply:
Noted.